### ISSUE TITLE
NVDA Is not reading the entire Paragraph

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#5188]
**Reported by Apoorva on 2015-06-30 04:29**
Hi Team,
    I noticed a weird behavior when i tested my page in Mozilla with NVDA. The reader does not read the whole paragraph at one stroke. I had to strike the NVDA +down arrow for two or more times to complete a paragraph. is it a default behavior?

**Comment by ehollig** [comment]
Any additional information from the creater of this issue? CC @jcsteh

**Comment by jcsteh** [comment]
Closing, as we don't have an easy way to find the reporter and this issue is now two years old without activity. We can reopen if the reporter provides the details requested.
